packages/block-library/src/image/styles.native.scss

```diff
@@ -1,5 +1,21 @@
 // @format
 
+.badge {
+	position: absolute;
+	top: 10px;
+	left: 10px;
+	background-color: rgba(85, 85, 85, 0.7);
```

Comment:
Shouldn't it be `rgba(26, 26, 26, 0.5)` as in [the spec](https://github.com/wordpress-mobile/gutenberg-mobile/issues/968#issue-442455454)?

Reply:
🤔 I don't think so, but I could be wrong.

I think that spec is superseded by [this one](https://zeroheight.com/226850145/p/11878e/b/793b21/i/6492732). Thomas Bishop said he'd be updating the spec [here](https://hogwartsp2.wordpress.com/2019/12/11/%f0%9f%91%8b%f0%9f%8f%bd-hello-im-matt/#comment-15895) and I believe that the one he mentions [here](https://a8c.slack.com/archives/C6UJ0KRKQ/p1576120839141100?thread_ts=1576119860.137300&cid=C6UJ0KRKQ) is the update he was referring to.

Reply:
I see, indeed I wasn't up to date 👌 

(btw you might want to use shortlinks to share internal urls not to leak any private data)